packages/pluggableWidgets/timeline-web/src/Timeline.editorConfig.ts

```diff
@@ -1,5 +1,13 @@
-import { hidePropertiesIn, hidePropertyIn, Problem, Properties } from "@widgets-resources/piw-utils";
+import {
+    hidePropertiesIn,
+    hidePropertyIn,
+    Problem,
+    Properties,
+    StructurePreviewProps
+} from "@widgets-resources/piw-utils";
 import { TimelinePreviewProps } from "../typings/TimelineProps";
+import { lineAndDotSVG } from "./preview/preview";
```

Comment:
You can import an svg file directly. Here you can see how this is done for floating-action-button-native:
```
import { ImageProps, ContainerProps } from "@widgets-resources/piw-utils";
import { FloatingActionButtonPreviewProps } from "../typings/FloatingActionButtonProps";
import StructurePreviewFloatingActionButtonPrimarySVG from "./assets/StructurePreviewFloatingActionButtonPrimary.svg";

const rowLayoutContainer: ContainerProps = { type: "Container", children: [] };
const rowLayoutImage: ImageProps = {
    type: "Image",
    document: decodeURIComponent(StructurePreviewFloatingActionButtonPrimarySVG.replace("data:image/svg+xml,", "")),
    width: 32,
    height: 32
};

export const getPreview = (values: FloatingActionButtonPreviewProps) => ({
    type: "RowLayout",
    borders: false,
    padding: 8,
    columnSize: "grow",
    children:
        values.horizontalPosition === "left"
            ? [rowLayoutImage]
            : values.horizontalPosition === "center"
            ? [rowLayoutContainer, rowLayoutImage, rowLayoutContainer]
            : [rowLayoutContainer, rowLayoutImage]
});
```

Reply:
It seems that the web configs don't recognise .svg files. I have kept the current typescript exported string literal. I found an example like this elsewhere in the repo. Are you okay with keeping this?

Reply:
It shouldn't be too hard. This PR from Wesley for native changed 4 files only: https://github.com/mendix/widgets-resources/pull/399/files.

Give it a try, so we don't pile up our future to do stack. If it turns out to be more difficult, let's make a new story for it.

Reply:
👍